src/components/NewLocationPage/LocationOverview/LocationOverview.tsx

```diff
@@ -75,6 +79,13 @@ const LocationOverview: React.FC<{
       </GridContainer>
       <MobileOnly>
         <ShareButton onClickShare={onClickShare} />
+        <ShareButtons
+          eventCategory={EventCategory.ENGAGEMENT}
+          shareUrl={shareUrl}
+          shareQuote={shareQuote}
+          region={region}
+          isLocationPageHeader={true}
+        />
```

Comment:
The first share button component needs to be replaced with the second share button. I just haven't removed the first one yet to better illustrate the issue. Currently, the second share button will not stretch in the same way as the first. @chasulin we can sync on this to resolve.